### PR TITLE
Update egg-icarus--dedicated

### PR DESCRIPTION
**Pull request by Ballaual**
adjusted wine version from staging to latest to fix server not starting properly
Fixing: https://github.com/parkervcp/eggs/issues/2457

### CODE REVIEW

**Review by QuintenQVD0 — APPROVED** [review]
LGTM!

**Comment by HerrSammyDE** [comment]
> > when setting password for the server is not working. and the server is open for all to join.
> 
> pls do not use closed issues/merges pls and open a new issue instead!
> 
> the password have to be set here home/container/Icarus/Saved/Config/WindowsServer/ServerSettings.ini
> 
> the server won't show up as locked in steam server list only ingame.

Do all settings have to be set there? Because even in the "Icarus/Saved/Config/ServerSettings.ini" the "MaxPlayers=4" are not accepted. I have already included MaxPlayers in the start command, but that doesn't help either

**Comment by HerrSammyDE** [comment]
Yes, it must also be set there.

That should already help:
```
WIN_FILE=/mnt/server/Icarus/Saved/Config/WindowsServer/ServerSettings.ini
if [ -f "$WIN_FILE" ]; then
echo "Windows Server Config already exist skipping"
else
echo "Windows Server Config does not yet exist, making one"
mkdir -p /mnt/server/Icarus/Saved/Config/WindowsServer/
cd /mnt/server/Icarus/Saved/Config/WindowsServer/
curl -sSL -o ServerSettings.ini https://raw.githubusercontent.com/RocketWerkz/IcarusDedicatedServer/main/ServerSettings.ini
fi
```

**Comment by QuintenQVD0** [comment]
> Yes, it must also be set there.
> 
> That should already help:
> 
> Configuration Files:
> 
> ```
> {
>     "Icarus/Saved/Config/WindowsServer/ServerSettings.ini": {
>         "parser": "file",
>         "find": {
>             "SessionName": "SessionName=\"{{env.SERVER_NAME}}\"",
>             "JoinPassword": "JoinPassword=\"{{env.SERVER_PASSWORD}}\"",
>             "MaxPlayers": "MaxPlayers=\"{{env.MAX_PLAYERS}}\"",
>             "AdminPassword": "AdminPassword=\"{{env.ADMIN_PASSWORD}}\""
>         }
>     }
> }
> ```
> 
> Install script:
> 
> ```
> WIN_FILE=/mnt/server/Icarus/Saved/Config/WindowsServer/ServerSettings.ini
> if [ -f "$WIN_FILE" ]; then
> echo "Windows Server Config already exist skipping"
> else
> echo "Windows Server Config does not yet exist, making one"
> mkdir -p /mnt/server/Icarus/Saved/Config/WindowsServer/
> cd /mnt/server/Icarus/Saved/Config/WindowsServer/
> curl -sSL -o ServerSettings.ini https://raw.githubusercontent.com/RocketWerkz/IcarusDedicatedServer/main/ServerSettings.ini
> fi
> ```

Whats the issue as I don't understand the problem?

**Comment by HerrSammyDE** [comment]
> > Yes, it must also be set there.
> > That should already help:
> > Configuration Files:
> > ```
> > {
> >     "Icarus/Saved/Config/WindowsServer/ServerSettings.ini": {
> >         "parser": "file",
> >         "find": {
> >             "SessionName": "SessionName=\"{{env.SERVER_NAME}}\"",
> >             "JoinPassword": "JoinPassword=\"{{env.SERVER_PASSWORD}}\"",
> >             "MaxPlayers": "MaxPlayers=\"{{env.MAX_PLAYERS}}\"",
> >             "AdminPassword": "AdminPassword=\"{{env.ADMIN_PASSWORD}}\""
> >         }
> >     }
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Install script:
> > ```
> > WIN_FILE=/mnt/server/Icarus/Saved/Config/WindowsServer/ServerSettings.ini
> > if [ -f "$WIN_FILE" ]; then
> > echo "Windows Server Config already exist skipping"
> > else
> > echo "Windows Server Config does not yet exist, making one"
> > mkdir -p /mnt/server/Icarus/Saved/Config/WindowsServer/
> > cd /mnt/server/Icarus/Saved/Config/WindowsServer/
> > curl -sSL -o ServerSettings.ini https://raw.githubusercontent.com/RocketWerkz/IcarusDedicatedServer/main/ServerSettings.ini
> > fi
> > ```
> 
> Whats the issue as I don't understand the problem?

In the current version of the Egg, the "ServerSettings.ini" file is created and used in the "Icarus/Saved/Config/" directory. However, the server settings, such as the number of slots or the server password, are not applied if they are configured in this directory.

Through my testing, I have found that the configuration settings need to be made in the "Icarus/Saved/Config/WindowsServer/" path instead for them to take effect in the game.

To summarize, the path "Icarus/Saved/Config/" is therefore actually superfluous.

**Comment by Ballaual** [comment]
> Through my testing, I have found that the configuration settings need to be made in the "Icarus/Saved/Config/WindowsServer/" path instead for them to take effect in the game.

isnt this what i already told here?
> the password have to be set here
> home/container/Icarus/Saved/Config/WindowsServer/ServerSettings.ini

so what are we talking about?
